.drone.yml

```diff
@@ -230,3 +230,34 @@ steps:
       VERSION: "${DRONE_TAG}"
     depends_on:
       - clone
+
+---
+kind: pipeline
+type: kubernetes
+name: migration deploy tag
+trigger:
+  event:
+    - tag
+  branch:
+    - main
+depends_on:
+  - build tag
```

Comment:
Might be worth consolidating this into the other tag build -- just makes it a bit easier to read? But both ways works fine.
```suggestion
```

Reply:
I've merged the migrator step into the first tag build.